examples/vpp-icmp/vppagent-client/cmd/vppagent.go

```diff
@@ -24,25 +26,32 @@ func CreateVppInterface(nscConnection *connection.Connection, baseDir, vppAgentE
 		grpc.WithStreamInterceptor(
 			otgrpc.OpenTracingStreamClientInterceptor(tracer)))
 
+	fullyQualifiedSocketFilename := path.Join(baseDir, nscConnection.GetMechanism().GetSocketFilename())
```

Comment:
Hmm not sure this code should precede the error check from the grpc.Dial. Can you please move the code back down?